modules/ROOT/pages/connected-apps-org-admin.adoc

```diff
@@ -2,37 +2,84 @@
 
 As an organization administrator at the root organization or business group level, you are the only user who can view and manage connected apps in Access Management. You can use authorization policies to dictate which apps are authorized to access user data, and you can add apps to an allowlist.
 
-== Owned Apps
+[[connected-app-ownership]]
+== Connected App Ownership
 
-To manage your connected apps, navigate to *Access Management* and select the *Connected Apps* tab. You can add apps in the *Owned Apps* section and manage app access in the *Authorizations* section.
-For more information about the *Owned Apps* section, see the documentation for xref:connected-apps-developers.adoc[app developers].
+Connected app ownership gets transferred to the root organization owner when the user who created the connected app:
+
+* is deleted.
```

Comment:
I think these should start with [capital letter](https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#:~:text=Capitalize%20the%20first%20letter%20of%20each%20item.) and have no period (even though they complete the intro sentence).